instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -395,9 +397,25 @@ def _instrument(self, **kwargs):
                 unit="switches",
             )
 
+        if "process.open_file_descriptor.count" in self._config:
+            self._meter.create_observable_up_down_counter(
+                name=f"process.{self._python_implementation}.open_file_descriptor.count",
```

Comment:
Taking a second look I don't think this is dependent on the python implementation though. Do we need it? Genuine question, no clue why we are adding these prefixes.

Reply:
Good question. If we look at [semconv here](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-semantic-conventions/src/opentelemetry/semconv/_incubating/metrics/process_metrics.py#L177) there's no reference to python implementation in the metric name.

Reply:
It does look like the original intent was to match the semantic conventions for [runtime environment metrics](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/runtime/README.md#metrics) where the runtime is prefixed. I believe the question is whether `process.*` metrics should be treated as runtime metrics or `process` metrics. To me it is mostly the latter so we should probably follow the semantic conventions for [process metrics](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/process-metrics.md#process-metrics) and remove the runtime prefix altogether.

Reply:
@emdneto 

For this pr, I suggest removing the prefix for file descriptor and maybe opening up an issue to remove the prefixes for everything else is people agree on this approach.

Reply:
I'll open the issue. Thanks